plugin/dnstap/handler.go

```diff
@@ -15,9 +15,11 @@ import (
 
 // Dnstap is the dnstap handler.
 type Dnstap struct {
-	Next plugin.Handler
-	IO   IORoutine
-	Pack bool
+	Next           plugin.Handler
+	IO             IORoutine
+	JoinRawMessage bool
```

Comment:
JoinRawMessage should probably have some docs explaining what it does when you set it.